nntrainer/layers/cl_layers/blas_kernels.cpp

```diff
@@ -115,7 +115,7 @@ void sgemv_cl(const float *matAdata, const float *vecXdata, float *vecYdata,
       break;
     }
 
-    result = kernel_sgemv.SetKernelArguments(4, &dim2, sizeof(int));
+    result = kernel_sgemv.SetKernelArguments(4, &lda, sizeof(int));
```

Comment:
Is this a bugfix? It doesn't look like an fp16 implementation.

Could you please separate bugfix commit and feature-implementation commit?
Mixing up the two topics in a single commit confuses reviewers.

Reply:
I have used `lda`  in SGEMV kernel to generalize it for future uses.